kge/dataset.py

```diff
@@ -117,6 +166,10 @@ def _load_triples(filename: str, delimiter="\t", use_pickle=False) -> Tensor:
     def load_triples(self, key: str) -> Tensor:
         "Load or return the triples with the specified key."
         if key not in self._triples:
+            if self.folder is None or not os.path.exists(self.folder):
```

Comment:
add a method `is_available` or so and move the code there. This way, we can add more checks later on. E.g., is_available() may check everything, `is_available(key)` may check an individual file etc.